<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceInfoSettingsBasic</class>
 <widget class="QDialog" name="DeviceInfoSettingsBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);</string>
  </property>
  <widget class="QLabel" name="deviceInfoSettingsTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1788C3;</string>
   </property>
   <property name="text">
    <string>Device Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backDeviceInfoSettings">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>28</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:3px;
background-color: rgb(70, 70, 73);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../rsc/resource.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollAreaDeviceInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>63</y>
     <width>480</width>
     <height>209</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QScrollBar:vertical {
     border: none;
     width: 5px;
 }
 QScrollBar::handle:vertical {
     background:#1788c3;
     min-height: 20px;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 2px #1788c3;
     width: 5px;
     height: 5px;
     background: #1788c3;
 }</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContentsDeviceInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>473</width>
      <height>207</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>481</width>
       <height>317</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutDeviceInfo">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QFrame" name="deviceNameDeviceInfo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(59, 59, 59);
border:1px;
border-radius:5px;</string>
        </property>
        <layout class="QHBoxLayout" name="monWidget_9">
         <item>
          <layout class="QVBoxLayout" name="deviceNameDeviceInfoLeftLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="deviceNameLeftTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 189, 189);</string>
             </property>
             <property name="text">
              <string>Device Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="deviceNameDeviceInfoRightLayout">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="deviceNameRightTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(33, 150, 243);</string>
             </property>
             <property name="text">
              <string>1234</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ruHoursDeviceInfo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(59, 59, 59);
border:1px;
border-radius:5px;</string>
        </property>
        <layout class="QHBoxLayout" name="monWidget_3">
         <item>
          <layout class="QVBoxLayout" name="ruHoursDeviceInfoLeftLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="ruHoursLeftTitleDeviceInfo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>10</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 189, 189);</string>
             </property>
             <property name="text">
              <string>Device Run Hours</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ruHoursLeftSubTitleDeviceInfo">
             <property name="font">
              <font>
               <family>Roboto</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(97, 97, 97);</string>
             </property>
             <property name="text">
              <string>Hours</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ruHoursDeviceInfoRightLayout">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="ruHoursRightTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(33, 150, 243);</string>
             </property>
             <property name="text">
              <string>5000</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="serialNumberVersionDeviceInfo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(59, 59, 59);
border:1px;
border-radius:5px;</string>
        </property>
        <layout class="QHBoxLayout" name="monWidget_8">
         <item>
          <layout class="QVBoxLayout" name="serialNumberDeviceInfoLeftLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="serialNumberLeftTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 189, 189);</string>
             </property>
             <property name="text">
              <string>Serial Number</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="serialNumberDeviceInfoRightLayout">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="serialNumberRightTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(33, 150, 243);</string>
             </property>
             <property name="text">
              <string>1234</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="hwVersionDeviceInfo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(59, 59, 59);
border:1px;
border-radius:5px;</string>
        </property>
        <layout class="QHBoxLayout" name="monWidget_7">
         <item>
          <layout class="QVBoxLayout" name="hwVersionDeviceInfoLeftLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="hwVersionLeftTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 189, 189);</string>
             </property>
             <property name="text">
              <string>HW Version</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="hwVersionDeviceInfoRightLayout">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="hwVersionRightTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(33, 150, 243);</string>
             </property>
             <property name="text">
              <string>1234</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fwVersionDeviceInfo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(59, 59, 59);
border:1px;
border-radius:5px;</string>
        </property>
        <layout class="QHBoxLayout" name="monWidget_4">
         <item>
          <layout class="QVBoxLayout" name="fwVersionDeviceInfoLeftLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="fwVersionLeftTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 189, 189);</string>
             </property>
             <property name="text">
              <string>FW Version</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="fwVersionDeviceInfoRightLayout">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="fwVersionRightTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(33, 150, 243);</string>
             </property>
             <property name="text">
              <string>1234</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="swVersionDeviceInfo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(59, 59, 59);
border:1px;
border-radius:5px;</string>
        </property>
        <layout class="QHBoxLayout" name="monWidget_6">
         <item>
          <layout class="QVBoxLayout" name="swVersionDeviceInfoLeftLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="swVersionLeftTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 189, 189);</string>
             </property>
             <property name="text">
              <string>SW Version</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="swVersionDeviceInfoRightLayout">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="swVersionRightTitleDeviceInfo">
             <property name="styleSheet">
              <string notr="true">color: rgb(33, 150, 243);</string>
             </property>
             <property name="text">
              <string>1234</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rsc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
